src/main/java/seedu/address/model/appointment/FilterAppointmentPredicate.java

```diff
@@ -6,22 +6,25 @@
  * Tests that a {@code Appointment}'s {@code startDateTime} and {@code endDateTime} matches the date times given.
  */
 public class FilterAppointmentPredicate implements Predicate<Appointment> {
-    private final StartDateTime startDateTime;
-    private final EndDateTime endDateTime;
+    private final StartDateTime lowerBoundDateTime;
```

Comment:
What's the reason behind changing the variable names here? I was quite okay with it previously. 

Note that this might affect further PRs, so avoid changing if possible

Reply:
start end date is part of the appointment
while lowe rbound is part of the predicate
this is just to see better which is which

Reply:
Hmm, I see, as long as it doesn't cause any refactoring issues I'm fine with it